array-manipulation-3/src/equal.js

```diff
@@ -0,0 +1,13 @@
+/* exported equal */
+
+function equal(first, second) {
+  var i = 0;
+  for (i; i < first.length; i++) {
+    for (i; i < second.length; i++) {
+      if (first[i] !== second[i]) {
+        return false;
+      }
+    }
+  }
```

Comment:
This can be simplified a bit. Try to handle the case where the arrays are not the same length before even starting your loop. Then you can assume that they are the same length and only use one loop.

Reply:
Done.